stoobly_agent/cli.py

```diff
@@ -50,6 +51,7 @@ def main(ctx):
 main.add_command(endpoint)
 main.add_command(feature)
 main.add_command(intercept)
+main.add_command(log)
```

Comment:
Thoughts on if we should make this a subcommand of `request` e.g. `stoobly-agent request log`

Reply:
Great idea!